setup.py

```diff
@@ -39,7 +39,7 @@
     ext_modules=[
         Extension("intbitset",
                   ["src/intbitset.c", "src/intbitset_impl.c"],
-                  extra_compile_args=['-O3', '-march=native']
```

Comment:
It was quite important for enabling vectorization in the past. Maybe @kaplun has more info about it.